dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/imports/bundle/ReportSummaryIntegrationTest.java

```diff
@@ -263,33 +263,4 @@ void testStatsCountForOneUpdateEventAndOneNewEvent() throws IOException {
     assertEquals(0, trackerImportEventReport.getStats().getIgnored());
     assertEquals(0, trackerImportEventReport.getStats().getDeleted());
   }
-
-  @Test
-  void testStatsCountForOneUpdateEventAndOneNewEventAndOneInvalidEvent() throws IOException {
```

Comment:
Not needed anymore?

Reply:
We cannot have invalid UID for event passed to the importer anymore